azure-pipelines/ui-automation/msal-broker-BrokerHost-nightly.yml

```diff
@@ -147,54 +129,15 @@ stages:
       - brokers
     displayName: Running MSAL with Broker Test UI Test Suite
     jobs:
-      - job: 'msal_with_broker'
-        displayName: Running MSAL with Broker Test Plan
-        timeoutInMinutes: 90
-        pool:
-          vmImage: ubuntu-latest
-        variables:
-          resultsDir: "msal-BrokerHost-$(Build.BuildId)-$(Build.BuildNumber)"
-        steps:
-          - checkout: none
-          - task: DownloadSecureFile@1
-            displayName: 'Download Firebase Service Account Key File'
-            name: gcServiceAccountKey
-            inputs:
-              secureFile: AndroidFirebaseServiceAccountKey.json
-              retryCount: 5
-          - download: current
-          - script: gcloud version
-            displayName: 'Check gcloud version'
-          - task: Bash@3
-            displayName: Run UI Automation on Firebase
-            inputs:
-              targetType: inline
-              script: |
-                gcloud auth activate-service-account --key-file "$(gcServiceAccountKey.secureFilePath)"
-                gcloud config set project "$(gCloudProjectId)"
-                gcloud firebase test android run \
-                  --type instrumentation \
-                  --app "$(Pipeline.Workspace)/msalautomationapks/$(msalAutomationAppApk)" \
-                  --test "$(Pipeline.Workspace)/msalautomationapks/$(msalAutomationAppTestApk)" \
-                  --device "model=${{ parameters.firebaseDeviceId }},version=${{ parameters.firebaseDeviceAndroidVersion }}" \
-                  --timeout "$(firebaseTimeout)" \
-                  --other-files \
-                  "/data/local/tmp/BrokerHost.apk=$(Pipeline.Workspace)/brokerHost/$(brokerHostApk),\
+      - template: ./templates/run-on-firebase.yml
+        parameters:
+          automationAppApkPath: "$(Pipeline.Workspace)/msalautomationapks/msalautomationapp-local-BrokerHost-debug.apk"
+          automationAppTestApkPath: "$(Pipeline.Workspace)/msalautomationapks/msalautomationapp-local-BrokerHost-debug-androidTest.apk"
+          testTargetPackages: ${{ parameters.testTargetPackages }}
```

Comment:
Where are the test targets actually being set now that we removed --test-targets from the script?

Reply:
They are still being supplied as pipeline parameters. Just coming from a higher level. The calling pipeline.